### PR TITLE
chore: new labels

### PR DESCRIPTION
**Pull request by jpvajda**
Closes https://github.com/newrelic/docs-website/issues/1096

This PR adds the preferred labels to the create issue button component

### CODE REVIEW

File: packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:46]: Just to verify, do we want this to apply to all sites? Or just the docs site?

**Review by jerelmiller — APPROVED** [review]
The change looks great! I had one question for you just to make sure we are updating this in the way we think we are.

File: packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js
Comment (jpvajda) [reply packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:46]: @jerelmiller ideally just docs site, as these  labels aren't used on developer / oss for tracking issues.  so if there is some IF / ELSE logic I can add here, I'm happy to do that.

Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:46]: Ok! In that case, I think we should convert this into a prop for this component so that each site can specify its own set of labels. I think it probably still makes sense to default to `['bug']` if the prop isn't defined. Here are the changes I would make:

```js
const CreateIssueButton = ({ labels = ['bug'] /* <-- add `labels` prop }) => {
  // ...
  return (
    <GitHubIssueButton
      // ...
      labels={labels} // <--- use props instead of hard-coding the value here
    />
}

CreateIssueButton.propTypes = {
  labels: PropTypes.arrayOf(PropTypes.string) // <--- add prop type for new labels
}
```

We'll want to update the README as well to list the new `labels` prop for this component.

I'm going to dismiss my review until this is in place so we can get this working.

**Review by jerelmiller — CHANGES_REQUESTED** [review]
Since we've identified that we want each site to be able to override the labels, I'm going to request those changes before approving this PR.

**Comment by jpvajda** [comment]
@jerelmiller cool it's on m `todo` list for today.

File: packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:42]: Rather than using a static query here, could we instead just make the `labels` a prop to this component?

Comment (jpvajda) [reply packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:42]: sure... though I may need a pointer on how do that approach. 😄

Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:42]: See [this comment](https://github.com/newrelic/gatsby-theme-newrelic/pull/311#discussion_r587991463). This should provide the instructions you need 🙂

Comment (jpvajda) [reply packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:42]: oh I didn't even see that... ok yeah let me change the approach. at least I got practice with using static queries.

**Comment by jpvajda** [comment]
@jerelmiller so next steps are: once this approved / merged we just set the prop type we want on each site?

**Comment by jerelmiller** [comment]
Thats correct!

File: packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js:30]:
```suggestion
  labels = ['bug'],
```

Last minor thing! This is the default value, which will kick in if the `label` prop is not provided when using the `CreateIssueButton` on a site. Since `feedback` and `feedback-issue` are docs specific, let's default this to `bug` for all other sites. We'll override this default on the docs site when this change is merged in. Other than this, the change looks great!

**Review by jerelmiller — APPROVED** [review]

**Comment by jerelmiller** [comment]
Thanks for updating!

**Comment by jpvajda** [comment]
@jerelmiller also, the other sites will need to be updated this theme version as well to use this new proptype.

**Comment by jerelmiller** [comment]
> use this new proptype

Yes and no 😄 . That default value (the `= ['bug']`) you have defined will be the default set of labels on any site that doesn't provide a `labels` prop to this component. This should maintain backwards compatibility with the other sites.

**Pull request merged**
